src/FOM/UserBundle/Resources/public/css/userlist.datatables.css

```diff
@@ -0,0 +1,86 @@
+.dataTables_filter label input {
```

Comment:
The userlist.datatables.css is only included when you open the userlist and should totally override the global datatables CSS definition thus !important is a possible way to go. 

yep, .tdsmall ain't a good name for a class omg

Please open an issue for the SCSS as well.